.github/workflows/ci_cd.yml

```diff
@@ -40,27 +42,18 @@ jobs:
       - name: Test with tox
         run: tox -e style
 
-  # docs-style:
+  # doc-style: #TO BE ADDED!!!
```

Comment:
Can we add this already?

Reply:
Ow sure, I'll do it
Thanks for the reminder 😄